mysql-test/suite/innodb/r/log_encrypt_1_mk.result

```diff
@@ -1,4 +1,6 @@
 SET GLOBAL innodb_redo_log_encrypt = MASTER_KEY;
+Warnings:
+Warning	13068	InnoDB: Can't set redo log tablespace to be encrypted.
```

Comment:
Can we add a bit more?
These messages  "Blah X failed" are not helpful to user.

it should be more like

"Blah X failed because of Y reason. Please check keyring plugin is loaded or not" Optionally some variable names can also be mentioned.

Reply:
We could... with even more refactoring. There are two limitations here:

* most methods only return a boolean status
* actual error could be several levels below in the call chain (in this case: log_write_encryption -> log_file_header_fill_encryption -> fsp_header_fill_encryption_info)

In end-user error cases, I would give it 99% that it's the missing keyring - but we don't know that for sure when we generate this error message.